kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -663,6 +668,14 @@ qrcode-image-generator:
         margin=1
     }
 
+    cloudstorage {
+      metadata.replace_absolute_path={{ cloudstorage_replace_absolute_path | default('false') }}
+      relative_path_prefix={{ cloudstorage_relative_path_prefix_dial }}
```

Comment:
@anilguptatarento Please add default value for cloudstorage_relative_path_prefix_dial

Reply:
Added the default in main.yml